src/engine/guess_toolset.bat

```diff
@@ -1,4 +1,5 @@
-@ECHO OFF
+@IF NOT DEFINED DEBUG_BATCH @ECHO OFF
```

Comment:
It goes well with https://github.com/boostorg/build/pull/179/files#diff-180360612c6b8c4ed830919bbb4dd459R14